model_analyzer/triton/server/server_local.py

```diff
@@ -67,10 +69,14 @@ def start(self):
                 triton_env['CUDA_VISIBLE_DEVICES'] = ','.join(
                     [visible_gpus[uuid] for uuid in self._gpus])
 
+            if self._log_path:
+                self._log_file = open(self._log_path, 'w+')
+            else:
+                self._log_file = StringIO()
             self._tritonserver_process = Popen(cmd,
-                                               start_new_session=True,
-                                               stdout=PIPE,
+                                               stdout=DEVNULL,
```

Comment:
When we output everything to `DEVNULL` when are we going to use the `self._log_file`?

Reply:
Yep thats leftover, will remove it.